source/assets/js/helpers/mediaqueries.js

```diff
@@ -17,10 +17,15 @@
  * }
```

Comment:
I'd remove the `Check the current breakpoint:` block above since we don't need it anymore.